service/src/main/java/uk/gov/hmcts/reform/fpl/model/CaseData.java

```diff
@@ -629,4 +629,12 @@ public boolean hasFutureHearing(List<Element<HearingBooking>> hearingBookings) {
     public List<Element<uk.gov.hmcts.reform.fpl.model.order.CaseManagementOrder>> getSealedCMOs() {
         return defaultIfNull(sealedCMOs, new ArrayList<>());
     }
+
+    @JsonIgnore
+    public HearingBooking getNextHearingAfterCmo(UUID cmoID) {
+        return unwrapElements(hearingDetails).stream()
+            .filter(hearingBooking -> !hearingBooking.getCaseManagementOrderId().equals(cmoID))
+            .min(comparing(HearingBooking::getStartDate))
+            .orElse(HearingBooking.builder().build());
+    }
```

Comment:
I don't see how this method work.. Let's assume I have got cmoID 6. And I have got following hearings:
1. Hearing with CMO ID 4, case management. Scheduled for 20/07/2020 
2. Heaaring with CMO ID 6, further case management Scheduled for 22/07/2020.
3. Heaaring without CMO, issue resolution hearring scheduled for 30/07/2020.

That code will find hearing 1. which is not correct is it?